js/validation.js

```diff
@@ -0,0 +1,20 @@
+//  Target the form and its children
+const contactForm = document.querySelector('#contact-form');
+const email = document.querySelector('#email');
+const errorMessage = document.querySelector('#error-msg');
+
+contactForm.addEventListener('submit', (e) => {
+  //  Prevent the form submit
+  e.preventDefault();
+  // Verify the email
+  const emailValue = email.value;
+  if (emailValue.toLowerCase() === emailValue) {
+    errorMessage.innerHTML = '';
+    //  Submit the form
+    contactForm.submit();
+    // Reset the form
+    contactForm.reset();
+  } else {
+    errorMessage.innerHTML = 'Please, enter your email in lower case. ( For example : stephanie@gmail.com)';
+  }
```

Comment:
- [ ] - [OPTIONAL ] Well done @kensteph, your validation works fine after all, but I have a few suggestions to raise that can help you improve the project later, it is about the error message, my critics in color are well placed to say that the red color is not really matching the correct design dear, of course, it is an error color but please reduce its brightness, and style a bit properly,

- [ ] - [OPTIONAL ] The second is: will the error message disappear after a certain time? and the answer is not, what? yes, the answer is no, you can add a simple method to make the process clean and user-friendly, making the error message get off after 2 or 3sec can be fine.

<img width="548" alt="1" src="https://user-images.githubusercontent.com/77843050/209013207-5777ee57-579c-42da-b1f1-410dc65e8128.png">